dev-tools/scripts/smokeTestRelease.py

```diff
@@ -170,11 +170,15 @@ def checkJARMetaData(desc, jarFile, gitRevision, version):
 
     if gitRevision != 'skip':
       # Make sure this matches the version and git revision we think we are releasing:
-      # TODO: LUCENE-7023: is it OK that Implementation-Version's value now spans two lines?
-      verifyRevision = 'Implementation-Version: %s %s' % (version, gitRevision)
-      if s.find(verifyRevision) == -1:
-        raise RuntimeError('%s is missing "%s" inside its META-INF/MANIFEST.MF (wrong git revision?)' % \
+      match = re.search("Implementation-Version: (.+\r\n .+)", s, re.MULTILINE)
```

Comment:
Will the MANIFEST line endings always be `CRLF` ? According to the spec it could be `CR LF | LF | CR`...

Reply:
I'm not sure if CRLF on my machine was a Mac setting or something that ant or java tooling was doing. Hence the request for Windows/Linux testing help. But I chatted with Tim and confirmed that at least _his_ machine was also producing CRLF, so this wouldn't be holding up the release. I guess the correct expression would be `(\r\n?|\r) ` and then would have to do something more intelligent with the line removal too.